src/gromo/modules/growing_module.py

```diff
@@ -423,9 +444,19 @@ def sum_in_features(self, with_bias: bool = False) -> int:
         """
         if with_bias:
             return np.sum(
-                [module.in_features + module.use_bias for module in self.previous_modules]
+                [
+                    module.in_features + module.use_bias
+                    for module in self.previous_modules
+                    if not isinstance(module, MergeGrowingModule)
+                ]
             )
-        return np.sum([module.in_features for module in self.previous_modules])
+        return np.sum(
+            [
+                module.in_features
+                for module in self.previous_modules
+                if not isinstance(module, MergeGrowingModule)
+            ]
+        )
```

Comment:
I think we do not need `numpy` for this:
```suggestion
            return sum(
                       module.in_features + module.use_bias
                       for module in self.previous_modules
                       if not isinstance(module, MergeGrowingModule)
            )
        return sum(
                module.in_features
                for module in self.previous_modules
                if not isinstance(module, MergeGrowingModule)
        )
```